lib/conditionals.md

```diff
@@ -1,12 +1,22 @@
 ---
 title: Conditionals
-instructor_notes: Feel free to re-organize the headings (or add/remove headings) below. We included the headings for your benefit, but it's 100% fine if you want to write your responses in some different structure.
+instructor_notes: Feel  free to re-organize the headings (or add/remove headings) below. We included the headings for your benefit, but it's 100% fine if you want to write your responses in some different structure.
 ---
 
 # What is a conditional?
 
-Replace me with your response...
+Conditionals are used in code to produce a predefined solution given a user input or 
```

Comment:
I'm having trouble understanding what you mean by "solutions". Feel free to illustrate your thoughts with some code examples if it helps to explain!